aws/resource_aws_instance.go

```diff
@@ -391,6 +398,13 @@ func resourceAwsInstance() *schema.Resource {
 							Optional: true,
 							Computed: true,
 							ForceNew: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								// Suppress diff if volume_type is not io1
+								i := strings.LastIndexByte(k, '.')
+								vt := k[:i+1] + "volume_type"
+								v := d.Get(vt).(string)
+								return strings.ToLower(v) != ec2.VolumeTypeIo1
+							},
```

Comment:
Can you make this a function so that both DiffSuppressFunc blocks can use it?

Thanks!

Reply:
@Ninir Done